dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/FATSuite.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
+ *
```

Comment:
Should the copyright year be updated?

Reply:
For this one line code change I think we can skip the copyright date change.